gbpservice/nfp/configurator/drivers/base/base_driver.py

```diff
@@ -39,23 +39,30 @@ def clear_routes(self, context, kwargs):
         return SUCCESS
 
     def configure_healthmonitor(self, context, kwargs):
-        return self._check_vm_health(kwargs)
+        ip = kwargs.get('mgmt_ip')
+        COMMAND = 'ping -c5 '+ip
+        return self._check_vm_health(COMMAND, kwargs)
```

Comment:
Arguments of _check_vm_health() function doesn't match between function invocation and definition. Is this code tested?